sympy/vector/parametricregion.py

```diff
@@ -135,22 +135,22 @@ def _(obj):
     return [ParametricRegion(obj.args)]
 
 
-@parametric_region_list.register(Curve)
+@parametric_region_list.register(Curve)  # type: ignore
 def _(obj):
     definition = obj.arbitrary_point(obj.parameter).args
     bounds = obj.limits
     return [ParametricRegion(definition, bounds)]
 
 
-@parametric_region_list.register(Ellipse)
+@parametric_region_list.register(Ellipse) # type: ignore
```

Comment:
PEP-8 recommends to put two spaces before the comment `#` character. Should we keep this convention?

Reply:
We could do. I'm not sure it matters too much